source/tasks/RunRunbook/RunRunbookV6/runRunbook.ts

```diff
@@ -0,0 +1,52 @@
+import { Client, CreateRunbookRunCommandV1, RunbookRunRepository, Logger, TenantRepository, EnvironmentRepository } from "@octopusdeploy/api-client";
+import os from "os";
+import { TaskWrapper } from "tasks/Utils/taskInput";
+
+export async function createRunbookRunFromInputs(client: Client, command: CreateRunbookRunCommandV1, task: TaskWrapper, logger: Logger): Promise<DeploymentResult[]> {
```

Comment:
This is a small thing, but I feel like this function could do with some debug logging